patching/config/src/index.ts

```diff
@@ -1,29 +1,11 @@
-import { type PatchFile, type PatchInfo } from '@pnpm/patching.types'
-
-export interface ExtendedPatchInfo extends PatchInfo {
-  key: string
-}
-
-export function getPatchInfo (
-  patchedDependencies: Record<string, PatchFile> | undefined,
-  pkgName: string,
-  pkgVersion: string
-): ExtendedPatchInfo | undefined {
-  if (!patchedDependencies) return undefined
-  const pkgNameAndVersion = `${pkgName}@${pkgVersion}`
-  if (patchedDependencies[pkgNameAndVersion]) {
-    return {
-      file: patchedDependencies[pkgNameAndVersion],
-      key: pkgNameAndVersion,
-      strict: true,
-    }
-  }
-  if (patchedDependencies[pkgName]) {
-    return {
-      file: patchedDependencies[pkgName],
-      key: pkgName,
-      strict: false,
-    }
-  }
-  return undefined
-}
+export {
+  type ExtendedPatchInfo,
+  type PatchFile,
+  type PatchInfo,
+  type PatchGroup,
+  type PatchGroupRecord,
+} from '@pnpm/patching.types'
+export { allPatchKeys } from './allPatchKeys'
+export { groupPatchedDependencies } from './groupPatchedDependencies'
+export { getPatchInfo } from './getPatchInfo'
+export { type VerifyPatchesOptions, verifyPatches } from './verifyPatches'
```

Comment:
check which of these functions are actually used in other packages. I see at least `allPatchKeys` is not used in other packages. So, it doesn't need to be public.

Reply:
All except `allPatchKeys` are used outside of this package. https://github.com/pnpm/pnpm/pull/9236/commits/c49563caefc9b890a86d4ed3c6acac8f71604331